app/models/post.rb

```diff
@@ -32,12 +34,16 @@ def tags_string
   def tags_string=(value)
     @tags_list = []
```

Comment:
It looks like this code is only used in `create_tags_from_tag_string`. Is that true? If so, it can be moved into `create_tags_from_tag_string`